tcp.go

```diff
@@ -11,19 +12,34 @@ import (
 	"github.com/prometheus/log"
 )
 
+func dialTCP(target string, ssl bool, timeout time.Duration) (net.Conn, error) {
+	dialer := &net.Dialer{Timeout: timeout}
+	if !ssl {
+		return dialer.Dial("tcp", target)
+	}
+	config := &tls.Config{InsecureSkipVerify: true}
```

Comment:
Not sure I understand what you mean by "common".

There are some comments in #23 about passing options from prometheus to blackbox_exporter using GET parameters to `/probe`. Is this how you suggest to set InsecureSkipVerify?